src/errors/InvalidCredentialsError.js

```diff
@@ -0,0 +1,9 @@
+export default class InvalidCredentialsError extends Error {
+  constructor (...args) {
+    if (args.length !== 0) {
```

Comment:
Hmm many lines for something simple at first sight, aren't they?

Let's take a look at the `Error` class constructor's signature (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error):
```
new Error([message[, fileName[, lineNumber]]])
```
You only need to provide a default value for `message`. You can do that in a simpler (and more readable) way ;)

Go on